.github/workflows/helm-promote.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           chmod 755 promote.sh
           # Promote Helm charts
           ./promote.sh orc8r-${MAGMA_VERSION}.tgz
-          ./promote.sh cwf-orc8r-${MAGMA_VERSION}.tgz
+          ./promote.sh cawg-orc8r-${MAGMA_VERSION}.tgz
```

Comment:
Shouldn't this be "cwag"?

Reply:
Yes.

Reply:
But after I checked the corresponding files in the artifactory to make sure the change is correct, I found that in contrast to the docker promotion the acronym was not changed  from `cwf` to `cwag`. So I have to revert this change.